### DISCUSSION TITLE
How to run Faiss1.7.1 based on Python2？

### DISCUSSION THREAD

**Discussion by Sweetheartman** [facebookresearch/faiss#2039] [Q&A]
The Faiss version used by the project is 1.6.1 and is based on Python2. After the project is iterated, Faiss 1.7.1 is needed, but when I install it with conda, I can only use it under Python3. Is there any other way to use it? Used under Python2.

**Comment by mausch** [comment]
https://github.com/facebookresearch/faiss/discussions/2232
